src/commands/openapi/upload.ts

```diff
@@ -29,6 +29,11 @@ export default class OpenAPIUploadCommand extends BaseCommand<typeof OpenAPIUplo
 
   static flags = {
     key: keyFlag,
+    overwrite: Flags.boolean({
```

Comment:
Can you rename this flag?

```suggestion
    'confirm-overwrite': Flags.boolean({
```